spec/factories/tulsa_city_inmates.rb

```diff
@@ -0,0 +1,22 @@
+FactoryBot.define do
+  factory :tulsa_city_inmate do
+    inmateId { 'MyString' }
+    firstName { 'MyString' }
+    middleName { 'MyString' }
+    lastName { 'MyString' }
+    DOB { 'MyString' }
+    height { 'MyString' }
+    weight { 'MyString' }
+    hairColor { 'MyString' }
+    eyeColor { 'MyString' }
+    race { 'MyString' }
+    gender { 'MyString' }
+    arrestDate { 'MyString' }
+    arrestingOfficer { 'MyString' }
+    arrestingAgency { 'MyString' }
+    bookingDateTime { 'MyString' }
+    courtDate { 'MyString' }
+    releasedDateTime { 'MyString' }
+    courtDivision { 'MyString' }
```

Comment:
Use Faker for these (and use the right type, so for date I think it's `Faker::Date`) 
see https://github.com/faker-ruby/faker